certbot/cli.py

```diff
@@ -1156,9 +1156,10 @@ def _create_subparsers(helpful):
                 " Currently --csr only works with the 'certonly' subcommand.")
     helpful.add("revoke",
                 "--reason", dest="reason",
-                choices=CaseInsensitiveList(constants.REVOCATION_REASONS.keys()),
+                choices=CaseInsensitiveList(sorted(constants.REVOCATION_REASONS,
```

Comment:
Oh. I'm assuming you wanted them in this order for when they are displayed in error output. I didn't realize they were used there.

This is fine then. Just need to fix test failures which I think are due to the reason I described [here](https://github.com/certbot/certbot/pull/4987#pullrequestreview-54217053).